### PR TITLE
Changed printfs to spdlogs for DriveQuickTurn

### PR DESCRIPTION
**Pull request by ngupta-22**

### CODE REVIEW

**Comment by sfujimoto22** [comment]
Looks good!

**Comment by ashanawaz** [comment]
Nice!

File: 2021/Robot2021/src/main/cpp/commands/DriveQuickTurn.cpp [hunk 1]
```diff
@@ -9,6 +9,7 @@
 // it from being updated in the future.
 
 #include "RobotContainer.h"
+#include "spdlog/spdlog.h"
```
Comment (connorworley) [inline 2021/Robot2021/src/main/cpp/commands/DriveQuickTurn.cpp:12]:
```suggestion
#include <spdlog/spdlog.h>

```

**Review by connorworley — APPROVED** [review]
Looks good, just be sure to change that include!

**Pull request merged**